packages/atlas/src/components/_nft/NftWidget/NftInfoItem.styles.ts

```diff
@@ -5,13 +5,20 @@ import { sizes } from '@/styles'
 
 import { SizeProps, sizeObj } from './NftWidget.styles'
 
-export const InfoItemContainer = styled.div<SizeProps>`
+export const InfoItemContainer = styled.div<SizeProps & { disableSecondary?: boolean }>`
   display: grid;
   gap: ${sizes(1)};
-  grid-template:
-    'label' min-content
-    'content' min-content
-    'secondary' min-content / 1fr;
+  grid-template: ${({ disableSecondary }) =>
+    disableSecondary
+      ? `
+  'label' min-content
+  'content' min-content / 1fr
+  `
+      : `
+  'label' min-content
+  'content' min-content
+  'secondary' min-content / 1fr
+  `};
```

Comment:
Something's weird is happening here with line breaks. Maybe we could have separate `css` blocks for both cases? This wold improve readability